package.json

```diff
@@ -1,20 +1,34 @@
 {
   "name": "preq",
-  "version": "0.5.6",
+  "version": "0.6.0",
```

Comment:
No functional change, so we can bump the version in a PR that brings new functionality

Reply:
Done @d00rman